src/shared/@types/store.ts

```diff
@@ -46,6 +46,10 @@ export interface IWalletState {
   accounts: IAccountState[]
 }
 
+export type TAccountWithWallet = IAccountState & {
+  wallet: IWalletState
```

Comment:
What do you think?

```typescript
export type TAccountWithWallet = IAccountState & {
  wallet: Omit<IWalletState, 'accounts'>
}
```

Reply:
I am going to remove this type, I am not using it

Reply:
'accounts' exists within `wallet`. Why should I omit it?

Reply:
Maybe it could be more performative.